### DISCUSSION TITLE
PJ - Quais atividades vocês costumam registrar suas empresas?

### DISCUSSION THREAD

**Discussion by dionarodrigues** [frontendbr/forum#106] [Discussão]
# Hey galera! Blz?

Estou passando por um problema e gostaria de saber se vocês também já passaram por isso: minha empresa foi aberta com uma atividade bem genérica que a minha contadora indicou: manutenção e reparos de computador. E onde eu trabalhava na época como PJ, nunca tinha tido problemas com isso.

No entanto, uma empresa que estou negociando para trabalhar agora não aceitou essa atividade e argumentou que precisa de outras atividades mais amplas e que tem mais a ver com desenvolvimento de sites/sistemas.

**Assim, quero saber quais são as atividades que suas empresas estão registradas?**

Valeu!

**Comment by viniciusdocanto** [comment]
Eu botei essa e Treinamento em informatica, porque não achei nenhuma que se encaixasse.

-Edit
Ah eu abri como MEI também então não tem muita opção mesmo.

**Comment by rickbenetti** [comment]
O problema é o simples nacional que não aceita a categoria de Web Design CNAE 6201-5/02 cujo o valor de NFe é de 19,50%.

Eu mesmo pra me enquadrar no Simples e reduzir os custos tive que mudar eu não posso mais emitir nota como Consultoria em TI( ou Hospedagem de sites (CNAE 6311-9/00), hoje minhas CNAES são:
- 02682 - Processamento de dados e congêneres;
- 02917 - Suporte ténico em informática, instalação,  configuração e manutenção de programas; e
- 05762 - Serviço de instrução, treinamento e avaliação de conhecimentos de qualquer natureza.

Aqui tem uma [lista e aliquotas](http://contabnet.com.br/simples-nacional-servicos-de-informatica/) pra quem trabalha com TI.

**Comment by dionarodrigues** [comment]
Valeu @viniciusdocanto  e @rkb81 

Nessa lista que você me passou, @rkb81 , encontrei essas atividades com alíquota de 6% e que parecem ser interessantes:

6311-9/00 - Tratamento de dados, provedores de serviços de aplicação e serviços de hospedagem na internet
6319-4/00 - Portais, provedores de conteúdo e outros serviços de informação na internet
6190-6/01 - Provedores de acesso às redes de comunicações

**Comment by felipefialho** [comment]
Só verifica se eles aceitam essas atividades. Se sim, manda ver ;)

**Comment by andrearanha** [comment]
Pessoal,

Eu sempre faturo como "Suporte técnico em informática", mas poderia ser "Análise e desenvolvimento de sistemas" numa boa, hoje. Na hora de abrir a empresa, a nomenclatura muda um pouco, mas pode ser aberta com serviços desse tipo.

A partir de **2018**, o Simples Nacional mudou nesse ponto.
Não só aceita atividades técnicas, como admite um imposto de 6% nelas. Porém, para pagar só 6% você precisa tirar um certo valor de pró-labore pra sua pessoa física. Se não fizer isso, aí paga 15,5%.

Mais detalhes aqui: https://www.contratopj.com.br/simples-nacional-2018-para-profissionais-pj/
Acho que a mudança foi pra melhor.

Abs.

**Comment by leomaurodesenv** [comment]
Se ainda tiverem dúvidas sobre, eu recentemente fiz um comentário sobre: https://github.com/frontendbr/forum/discussions/602#discussioncomment-11755611
